packages/java-shell/src/test/resources/db/createCollectionWithTimeseries.js

```diff
@@ -7,18 +7,18 @@ db.createCollection('coll', {
         metaField: "metadata",
         granularity: "minutes"
     },
-    expireAfterSeconds: 86400
+    expireAfterSeconds: 60 * 60 * 24 * 365 * 10
 })
 // command
 db.coll.insertOne(
     {
         "metadata": { "sensorId": 5578, "type": "temperature" },
-        "timestamp": ISODate("2021-05-18T00:00:00.000Z"),
+        "timestamp": new ISODate("2022-09-09T00:00:00.000Z"),
         "temp": 12
     });
 // command
 db.coll.findOne({
-    "timestamp": ISODate("2021-05-18T00:00:00.000Z")
+    "timestamp": new ISODate("2022-09-09T00:00:00.000Z")
```

Comment:
Are these changes necessary? They seem okay but shouldn’t make a difference since `ISODate` is not a constructor

Reply:
Not necessary but just looks better to me